### PR TITLE
Add APIs to evaluate policies get result from OPA

### PR DESCRIPTION
**Pull request by baixinsui**

### CODE REVIEW

File: server/handle_policy_eval.go [hunk 1]
```diff
@@ -36,8 +36,8 @@ type evalResult struct {
 	IsSuccessful bool   `json:"isSuccessful" binding:"required"`
 }
 
-// @Tags			Policies Evaluate
-// @Summary		Evaluate the policies
+// @Tags			Policies Evaluate Input
```
Comment (iskey) [inline server/handle_policy_eval.go:38]:
```suggestion
// @Tags			Policies Evaluation
```

File: server/handle_policy_eval.go [hunk 2]
```diff
@@ -36,8 +36,8 @@ type evalResult struct {
...
+// @Summary		Evaluate input by policies
```
Comment (iskey) [inline server/handle_policy_eval.go:39]:
```suggestion
// @Summary		Evaluate the input by policies
```

File: server/handle_policy_eval.go [hunk 3]
```diff
@@ -80,8 +80,45 @@ func policiesEvaluateHandler(_ *config.Conf) gin.HandlerFunc {
 	}
 }
 
-// @Tags			Policies Evaluate
-// @Summary		Evaluate the policy
+// @Tags			Policies Evaluate Input
```
Comment (iskey) [inline server/handle_policy_eval.go:82]:
```suggestion
// @Tags			Policies Evaluation
```

File: server/handle_policy_eval.go [hunk 4]
```diff
@@ -113,13 +150,51 @@ func policyEvaluateHandler(_ *config.Conf) gin.HandlerFunc {
 	}
 }
 
+// @Tags			Policies Evaluate Input
```
Comment (iskey) [inline server/handle_policy_eval.go:153]:
```suggestion
// @Tags			Policies Evaluation
```

File: server/handle_policy_eval.go [hunk 5]
```diff
@@ -113,13 +150,51 @@ func policyEvaluateHandler(_ *config.Conf) gin.HandlerFunc {
...
+// @Summary		Evaluate input by policy
```
Comment (iskey) [inline server/handle_policy_eval.go:154]:
```suggestion
// @Summary		Evaluate the input by policies
```

File: server/handle_policy_eval.go [hunk 6]
```diff
@@ -176,3 +251,37 @@ func PolicyEval(policyRego string, policyQuery string, input interface{}) (decis
 
 	return true, nil
 }
+
+func PolicyEvalGetRawResult(policyRego string, input interface{}) (results rego.ResultSet, err error) {
```
Comment (iskey) [inline server/handle_policy_eval.go:255]: we can extract a common function from the  `PolicyEvalGetRawResult` and the `PolicyEval`.

**Review by iskey — CHANGES_REQUESTED** [review]
almost looks good, with only some minor comments

File: server/handle_policy_eval.go (hunk 6 above)
Comment (baixinsui) [reply server/handle_policy_eval.go]: Done.

**Review by swaroopar — APPROVED** [review]
LGTM

**Review by iskey — APPROVED** [review]
LGTM

**Pull request merged**